FAQs/README.md

```diff
@@ -338,3 +341,50 @@ stages:
         azurePowerShellVersion: 'LatestVersion'
 ```
 
+- ### **UI**
+
+### **1. I am not able to see subscriptions in AzTS UI. Are there any prequisites or any settings that I am missing?**
+If you are not able to see subscriptions in AzTS UI , please check below prerequisites followed by troubleshooting steps:
+
+### Prerequisites:
+1. Signed in user must have one of the following permissions at subscription or resource group scope: Owner, Contributor, ServiceAdministrator, CoAdministrator, AccountAdministrator, Security Reader, Security Admin.
```

Comment:
Pls. change this to a list for easy reading:

To access AzTS, the signed-in user must have one of the following permissions at the subscription or resource group level:

Owner
Contributor
Service Administrator
Co-Administrator
Account Administrator
Security Reader
Security Administrator
Please ensure that your user account has one of these permissions set up at the appropriate scope to effectively use AzTS.

Reply:
Updated